components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m

```diff
@@ -15,6 +15,7 @@
 #import "MDCActionSheetItemTableViewCell.h"
 
 #import <MaterialComponents/MaterialTypography.h>
+#import "MaterialRipple.h"
```

Comment:
Please use the same framework-style imports as above here to be self-consistent.